src/lib/components/ChatView/ChatView.svelte

```diff
@@ -136,8 +136,9 @@
       const response = await fetch('/api/messages', {
         method: 'POST',
         headers: {
-          'Content-Type': 'application/json',
           Accept: 'text/event-stream',
+          'Content-Type': 'application/json',
```

Comment:
Do I need to add the CSRF token in the headers? I think I can get the CSRF token from the session, which is  handled by the cookies sent by the client in every request.

Reply:
Yes, we need to add CSRF token in the headers.